packages/babel-helper-validator-identifier/src/identifier.ts

```diff
@@ -52,7 +52,7 @@ function isInAstralSet(code: number, set: readonly number[]): boolean {
 // Test whether a given character code starts an identifier.
 
 export function isIdentifierStart(code: number): boolean {
-  if (code < charCodes.uppercaseA) return code === charCodes.dollarSign;
+  if (code === charCodes.dollarSign) return true;
```

Comment:
Thank you for the PR. Unfortunately the logic here are changed. 

Take `,` (44) for example, previously `isIdentifierStart` returns `false` because `,` (44) is less than `A` (65) and it is not `$`. Now it will return true because the first branch is skipped and `,` (44) is of course less than `Z` (90). 

Therefore the CI is failing.

Reply:
Thank you for your guidance.